integration-tests/emulated/common/Cargo.toml

```diff
@@ -0,0 +1,81 @@
+[package]
+name = "integration-tests-common"
+version = "1.0.0"
+authors.workspace = true
+edition.workspace = true
+license = "Apache-2.0"
+description = "Common resources for integration testing with xcm-emulator"
```

Comment:
After bumping repo to sdk v1.4 or v1.5 release, we can drop 90% of the code in this module and get it from https://crates.io/crates/emulated-integration-tests-common/1.0.0